### DISCUSSION TITLE
FIlter ability

### DISCUSSION THREAD

**Discussion by ZatlaTN** [akumidv/tradingview-assistant-chrome-extension#161] [Ideas]
Hi can you add us a way to set more then 1 filter like multiple filters
![image](https://github.com/akumidv/tradingview-assistant-chrome-extension/assets/63630464/91318331-e412-452c-8e0f-e27dbe5cfefe)

**Comment by dymasty** [comment]
second filter is really needed

**Comment by akumidv** [comment]
Yes, this is will be in new major version
